CHANGELOG.md

```diff
@@ -25,6 +26,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 
 [Unreleased]: https://github.com/smart-leds-rs/smart-leds-trait/compare/v0.2.0...HEAD
```

Comment:
```suggestion
[Unreleased]: https://github.com/smart-leds-rs/smart-leds-trait/compare/v0.2.1...HEAD
```

Reply:
Argh, it's always the most trivial stuff. Thanks for the review!